libs/langchain/langchain/utilities/sql_database.py

```diff
@@ -50,6 +50,7 @@ def __init__(
         custom_table_info: Optional[dict] = None,
         view_support: bool = False,
         max_string_length: int = 300,
+        harmful_keywords: Optional[dict] = None,
```

Comment:
nit: are these "harmful" or just "restricted"/"protected"? maybe restricted_keywords would be a slightly more intuitive name?

Reply:
Indeed it would be less "agressive" and more coherent to reality